### PR TITLE
add welcome  bot 

### PR DESCRIPTION
**Pull request by shivam-sharma7**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
review/ @MonalikaPatnaik

File: .github/workflows/welcome.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            👋 @{{ author }}
+             Thanks for opening your first issue here! Be sure to follow the issue template!
+
+          FIRST_PR: |
+            👋 @{{ author }}
+            Thanks for opening this pull request! Please check out our contributing guidelines.
+
+          FIRST_PR_MERGED: |
+            🎉 @{{ author }}
+            Congrats on merging your first pull request! We here at behaviorbot are proud of you!
```
Comment (MonalikaPatnaik) [inline .github/workflows/welcome.yml:23]: We here at behaviorbot?
Change it to text like-- We appreciate your contribution in UMatter! or something like that?

File: .github/workflows/welcome.yml [hunk 2]
```diff
@@ -12,12 +12,12 @@ jobs:
         with:
           FIRST_ISSUE: |
             👋 @{{ author }}
-             Thanks for opening your first issue here! Be sure to follow the issue template!
+             We appreciate your contribution in UMatter! or something like that?
```
Comment (MonalikaPatnaik) [inline .github/workflows/welcome.yml]: I didn't told you to change this..

File: .github/workflows/welcome.yml [hunk 3]
```diff
@@ -12,12 +12,12 @@ jobs:
...
 
           FIRST_PR: |
             👋 @{{ author }}
             Thanks for opening this pull request! Please check out our contributing guidelines.
 
           FIRST_PR_MERGED: |
             🎉 @{{ author }}
-            Congrats on merging your first pull request! We here at behaviorbot are proud of you!
\ No newline at end of file
+            Congrats on merging your first pull request! We here at behaviorbot are proud of you!
```
Comment (MonalikaPatnaik) [inline .github/workflows/welcome.yml]: Replace this-- We here at behaviorbot are proud of you!
to: We appreciate your contribution in UMatter!

File: .github/workflows/welcome.yml (hunk 2 above)
Comment (shivam-sharma7) [reply .github/workflows/welcome.yml]: resolved @MonalikaPatnaik

File: .github/workflows/welcome.yml (hunk 3 above)
Comment (shivam-sharma7) [reply .github/workflows/welcome.yml]: resolved All @MonalikaPatnaik

**Pull request merged**